plugins/woocommerce/src/Admin/API/Reports/GenericStatsController.php

```diff
@@ -4,19 +4,57 @@
 defined( 'ABSPATH' ) || exit;
 
 use Automattic\WooCommerce\Admin\API\Reports\GenericController;
-use Automattic\WooCommerce\Admin\API\Reports\GenericQuery;
 
 /**
- * Generic base for all Stats controllers.
+ * Generic base for all stats controllers.
+ *
+ * {@see GenericController Generic Controller} extended to be shared as a generic base for all Analytics stats controllers.
+ *
+ * Besides the `GenericController` functionality, it adds conventional stats-specific collection params and item schema.
+ * So, you may want to extend only your report-specific {@see get_item_properties_schema() get_item_properties_schema()}`.
+ * It also uses the stats-specific {@see get_items() get_items()} method,
```

Comment:
Can you clarify how these see tags are formatted? If I understand it correctly it should be `@see URL <description>`. If the description is optional then do we need to repeat it if it's the same as the object we are linking to?

Also is the `@see` for internal and `@link` for external? Seems like there is a lot of formatting here in the docblocks, but I'd find it a bit overwhelming if I'd have to maintain that.

Reply:
Theoretically, this should be `@see URL|NodeId <description>`.
But I didn't manage to make it work like so with our code-reference generator. It does not create a link at all, plus, for some reason, if you do not provide the description, it does not render anything.

So, to avoid spending too much time fixing the code-reference generator, I used this workaround to explicitly add descriptions everywhere.

----
> Also is the @see for internal and @link for external? 

Intuitively to me `@see` would be a reference to something related, while `@link` for link of any kind. However, according to phpDocumentor: 

`@link` https://docs.phpdoc.org/2.9/references/phpdoc/inline-tags/link.html
> The effects of the inline version of this tag are not yet fully implemented in PhpDocumentor2. There’s only URI support (i.e. no support for [Structural Elements](https://docs.phpdoc.org/2.9/glossary.html#term-structural-elements)), and even that is available only in long descriptions.

So, it is simply not usable here

`@see` https://docs.phpdoc.org/2.9/references/phpdoc/inline-tags/see.html
> The @see tag can be used to define a reference to other [Structural Elements](https://docs.phpdoc.org/2.9/glossary.html#term-structural-elements) or to an URI.

Reply:
In the long run. I was thinking of fixing our `code-reference` generator to actually support just `{@see GenericController}` syntax, but that's beyond the scope of cleaning up analytics.